README.md

```diff
@@ -60,6 +62,46 @@ pro::proxy<poly::Drawable> CreateRectangleAsDrawable(int width, int height) {
 }
 ```
 
+Here is another demo showing how to define overloads in a dispatch. Note that `.invoke<>` can be ommitted when only 1 dispatch is defined in a facade:
+
+```cpp
+// Abstraction (poly is short for polymorphism)
+namespace poly {
+
+PRO_DEF_MEMBER_DISPATCH(Log, Log,
```

Comment:
nit: use different names to distinguish between dispatch and implementation?

Reply:
Removed the second parameter